### ISSUE TITLE
ordered-imports: missing ImportEqualsDeclaration

### ISSUE THREAD

**Issue by ikatyang** [palantir/tslint#3043]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.4.1
- __Running TSLint via__: CLI / VSCode

#### TypeScript code being linted

```ts
import c = require('./c'); // should error
import * as a from './a';
import b from './b';
```

with `tslint.json` configuration:

```json
{"rules": {"ordered-imports": [true]}}
```

#### Actual behavior
no error

#### Expected behavior
error on `c`

**Comment by adidahiya** [comment]
this was probably not implemented because import-equals statements are banned by default in the built-in configuration and they're usually treated specially (put in a separate import group). but it's a worthy enhancement we could implement.
